api/manifest.proto

```diff
@@ -46,34 +46,55 @@ message Flavor {
    map <string, Version> versions = 5;
 }
 
+
+enum Compliance {
+   FIPS_1402 = 0;
+}
+
 message Version {
    // This duplicated in order to make version easier to sort
    string name = 1;
-   // Key is operating system
-   // Note: Keys in proto maps cannot be enums, so this has to be a string.
-   map <string, OperatingSystem> operating_systems = 2; 
-}
-
-message OperatingSystem {
-   // Matches the key, duplicated here to be explicit.
-   OperatingSystemName name = 1;
-   
-   repeated Build builds = 2;
+   repeated Build builds = 2; 
 }
 
 message Build {
-   repeated string operating_system_versions = 1;
+   OperatingSystemFamily operating_system_family = 1;
    string download_location_url = 2;
 }
 
-enum Compliance {
-   FIPS_1402 = 0;
-}
+// Build maps 1:1 with Family
+// Family maps 1:m with Distribution
+// Distribution maps 1:1 with Version
+//
+//
+//         +-> UBUNTU -> 16.04+ LTS
+//         |
+// DEBIAN -|           
+//         |
+//         +-> DEBIAN -> 8+
+//
+//
+// Note: Distribution and version are considered metadata and should not be 
+// used as an exhaustive list or searched for via the CLI.
+message OperatingSystemFamily {
+   enum Name {
+      DEBIAN = 0;
+      RHEL = 1;
+      DARWIN = 2;
+      WINDOWS = 3;
```

Comment:
nit: No distribution for windows? On purpose?

Reply:
No, will add.